src/handle_form.inl

```diff
@@ -275,6 +279,7 @@ mg_handle_form_request(struct mg_connection *conn,
 					field_storage = MG_FORM_FIELD_STORAGE_SKIP;
 				}
 			}
+#if !defined(NO_FILESYSTEMS)
```

Comment:
Can we get rid of the definition of `MG_FORM_FIELD_STORAGE_STORE` if NO_FILESYSTEMS is defined?
(In addition to doing these ifdef)

Reply:
Will do